docs/integrating_with_existing_webapp.md

```diff
@@ -36,7 +36,7 @@ The structure of the oidc-sample-app we are configuring would be as follows:
         errorPage=
         logoutURL=logout
         callBackURL=http://localhost:8080/oidc-sample-app/oauth2client
-        scope=openid,internal_application_mgt_view
+        scope=openid,address,email,internal_application_mgt_view
```

Comment:
Btw i was wondering whether we need to request `internal_application_mgt_view` at all?

Reply:
@ChamathNS

Reply:
We ported the same params from our existing OIDC web apps (pickup dispatch, manager, etc).
This scope was added along with that IINM. I also think we don't need this one.
+1 for removing

Reply:
Address with commit ef0556e58d4730c07bf43454d7ba242614ccd2a1